configs/GEPRCF745_BT_HD/config.h

```diff
@@ -35,6 +35,10 @@
 #define USE_BARO
 #define USE_BARO_BMP280
 #define USE_BARO_DPS310
+#define USE_BARO_MS5611
+#define USE_MAG
+#define USE_MAG_QMC5883
+#define USE_MAG_IST8310
```

Comment:
If compass is not physical onboard it should be user option.

```suggestion
#ifndef USE_MAG
#define USE_MAG
#define USE_MAG_QMC5883
#define USE_MAG_IST8310
#endif
```